### PR TITLE
CP-37734: Replace negative language with "other pool member"

### PR DESCRIPTION
**Pull request by danilo-delbusso**
Entire solution only contains visible strings of the word in Messages.resx and FriendlyErrorNames.resx

Latter has been excluded.

### CODE REVIEW

**Review by CitrixChris — APPROVED** [review]

File: XenModel/Messages.resx [hunk 1]
```diff
@@ -2194,7 +2194,7 @@ To continue, disable HA, delete the bond, and then enable HA again.</value>
   <data name="BOND_DELETE_WILL_DISTURB_BOTH" xml:space="preserve">
     <value>{0}
 
-Deleting this bond will automatically transfer the management and secondary interfaces on the bond to the first slave member of the bond: 
+Deleting this bond will automatically transfer the management and secondary interfaces on the bond to the first other pool member of the bond:
```
Comment (danilo-delbusso) [inline XenModel/Messages.resx:2197]: First of the bonded members

File: XenModel/Messages.resx [hunk 2]
```diff
@@ -2205,7 +2205,7 @@ Deleting this bond will automatically transfer the management and secondary inte
   <data name="BOND_DELETE_WILL_DISTURB_PRIMARY" xml:space="preserve">
     <value>{0}
 
-Deleting this bond will automatically transfer the management interface on the bond to the first slave member of the bond: 
+Deleting this bond will automatically transfer the management interface on the bond to the first other pool member of the bond:
```
Comment (danilo-delbusso) [inline XenModel/Messages.resx:2208]: same as above

File: XenModel/Messages.resx [hunk 3]
```diff
@@ -8460,7 +8460,7 @@ Do you want to continue?</value>
 Do you want to continue?</value>
   </data>
   <data name="MESSAGEBOX_SLAVES_EJECT" xml:space="preserve">
-    <value>You must eject all slaves from the pool before you can delete the pool.</value>
+    <value>You must eject all other pool members from the pool before you can delete the pool.</value>
```
Comment (danilo-delbusso) [inline XenModel/Messages.resx:8463]: pool -> it (last word)

File: XenModel/Messages.resx [hunk 4]
```diff
@@ -8840,7 +8840,7 @@ You should only proceed if you have verified that these settings are correct.</v
     <value>Pooling is restricted with this server's license</value>
   </data>
   <data name="NEWPOOL_SLAVE_ROLE" xml:space="preserve">
-    <value>Your current role on the slave is not authorized to add the slave to a pool</value>
+    <value>Your current role on the other pool member is not authorized to add it to a pool</value>
```
Comment (danilo-delbusso) [inline XenModel/Messages.resx:8843]: Your current role *on this server* is not authorized to add it to a pool

File: XenModel/Messages.resx [hunk 5]
```diff
@@ -9621,7 +9621,7 @@ It is strongly recommended that you Cancel and apply the latest version of the p
     <value>{0} (Hidden)</value>
   </data>
   <data name="NIC_SLAVE" xml:space="preserve">
-    <value>{0} (Slave)</value>
+    <value>{0} (Other pool member)</value>
```
Comment (danilo-delbusso) [inline XenModel/Messages.resx:9624]: Bonded member

**Review by CitrixChris — APPROVED** [review]

File: XenModel/Messages.resx [hunk 6]
```diff
@@ -2214,7 +2214,7 @@ Deleting this bond will automatically transfer the management interface on the b
   <data name="BOND_DELETE_WILL_DISTURB_SECONDARY" xml:space="preserve">
     <value>{0}
 
-Deleting this bond will disrupt traffic through the secondary interface on the bond while the interface is moved to the first slave of the bond.</value>
+Deleting this bond will disrupt traffic through the secondary interface on the bond while the interface is moved to the first other pool member of the bond.</value>
```
Comment (kc284) [inline XenModel/Messages.resx:2217]: Apologies, this was skipped accidentally during the first review: "is moved to the first of the bonded members"

**Pull request closed without merging**